File: engine/model/src/main/scala/tools/packet/monster/carnival/MonsterCarnivalPartyPoints.scala
package tools.packet.monster.carnival

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class MonsterCarnivalPartyPoints(private var _team: Int, private var _currentPoints: Int, private var _totalPoints: Int) extends PacketInput {
  def team: Int = _team

  def currentPoints: Int = _currentPoints

  def totalPoints: Int = _totalPoints

  override def opcode(): SendOpcode = SendOpcode.MONSTER_CARNIVAL_PARTY_CP
}

File: engine/engine-model/src/main/java/tools/packet/monster/carnival/MonsterCarnivalPartyPoints.java
package tools.packet.monster.carnival;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record MonsterCarnivalPartyPoints(int team, int currentPoints, int totalPoints) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.MONSTER_CARNIVAL_PARTY_CP;
   }
}